realm/realm-library/src/androidTest/java/io/realm/RealmAnnotationTests.java

```diff
@@ -227,6 +228,63 @@ public void primaryKey_isIndexed() {
         assertTrue(table.hasSearchIndex(table.getColumnIndex("id")));
     }
 
+    @Test
```

Comment:
Would it be fine to remove tests unrelated to migration then? I think that way, we can see how this PR affects migration process clearly.